src/sage/features/databases.py

```diff
@@ -44,22 +57,89 @@ class DatabaseCremona(StaticFile):
         sage: DatabaseCremona().is_present()                                    # optional - database_cremona_ellcurve
         FeatureTestResult('database_cremona_ellcurve', True)
     """
-    def __init__(self, name="cremona", spkg="database_cremona_ellcurve"):
+    def __init__(self, name="cremona"):
         r"""
         TESTS::
 
             sage: from sage.features.databases import DatabaseCremona
             sage: isinstance(DatabaseCremona(), DatabaseCremona)
             True
         """
+        from sage.env import CREMONA_MINI_DATA_DIR, CREMONA_LARGE_DATA_DIR
+        CREMONA_DATA_DIRS = set([CREMONA_MINI_DATA_DIR, CREMONA_LARGE_DATA_DIR])
+        CREMONA_DATA_DIRS.discard(None)
+        search_path = CREMONA_DATA_DIRS or sage_data_path("cremona")
+
+        spkg = "database_cremona_ellcurve"
+        if name == 'cremona_mini':
+            spkg = "elliptic_curves"
+
         StaticFile.__init__(self, f"database_{name}_ellcurve",
-                            filename='{}.db'.format(name.replace(' ', '_')),
-                            search_path=CREMONA_DATA_DIRS,
+                            filename=f"{name}.db",
+                            search_path=search_path,
                             spkg=spkg,
                             url="https://github.com/JohnCremona/ecdata",
                             description="Cremona's database of elliptic curves")
 
 
+class DatabaseEllcurves(StaticFile):
+    r"""
+    A :class:`~sage.features.Feature` which describes the presence of
+    William Stein's database of interesting curves.
+
+    EXAMPLES::
+
+        sage: from sage.features.databases import DatabaseEllcurves
+        sage: bool(DatabaseEllcurves().is_present())  # optional - database_ellcurves
+        True
+    """
+    def __init__(self):
+        r"""
+        TESTS::
+
+            sage: from sage.features.databases import DatabaseEllcurves
+            sage: isinstance(DatabaseEllcurves(), DatabaseEllcurves)
+            True
+        """
+        from sage.env import ELLCURVE_DATA_DIR
+        search_path = ELLCURVE_DATA_DIR or sage_data_path("ellcurves")
+
+        StaticFile.__init__(self, "database_ellcurves",
+                            filename='rank0',
+                            search_path=search_path,
+                            spkg="elliptic_curves",
```

Comment:
```suggestion
                            spkg="elliptic_curves",
                            type='standard',
```

Reply:
`type=standard` is a sage-the-distro thing. Why do we need to add this information here as well? What is it useful for here?

Reply:
It suppresses the warning that you complained about

Reply:
Sure, but it seems unnecessary to repeat this information here. In the end, all that matters at runtime is whether the feature `is_present()` or not. Sad consolation that a feature `is_standard()` if it's not present.